### ISSUE TITLE
Error after wake-up: Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by Gappa** [digitalcreations/MaxTo#743]
**Describe the bug**
There is a .NET error dialog shown after waking up the computer from hibernation.

```
************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at MaxTo.Core.Native.MaxToWindowOperations.MaximizeToRegion(Rectangle region)
   at MaxTo.Companion.MessageHandlers.MaximizeWindowMessageHandler.ActInternal(String name, Message message)
   at MaxTo.Companion.MessageHandlers.WindowMessageHandlerBase.Act(Message message)
   at MaxTo.Companion.MessageHandlers.WindowMessageHandlerDispatcher.Dispatch(Message message)
   at MaxTo.Companion.Hook.HookController.MessageSourceOnWindowMessage(Object sender, Message message)
   at MaxTo.Core.Native.HiddenWindowMessageSource.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**To Reproduce**
1. Hibernate the computer
2. Wake it up

However, it only manifests on my notebook, desktop is fine.

**Expected behavior**
No error is shown.

**System information:**
- Windows version: Windows Pro 21H1 (19043.1081)
- MaxTo version: 2.2.0

**Additional context**
- [error.log](https://github.com/digitalcreations/MaxTo/files/6730754/error.log)
- ![image](https://user-images.githubusercontent.com/749981/124064422-065fde00-da35-11eb-822e-74986f9265c3.png)

**Comment by vegardlarsen** [comment]
Reproducing this seems difficult, but it seems possible to figure out. :)

**Comment by vegardlarsen** [comment]
This seems to have been the same issue as #753. Closing as a duplicate. Also, it will be fixed in version 2.2.1.
